query/src/main/java/io/keen/client/java/AbsoluteTimeframe.java

```diff
@@ -62,4 +62,12 @@ public AbsoluteTimeframe(String start, String end) {
 
         return timeframe;
     }
+
+    @Override
+    public String toString() {
```

Comment:
the way I ended up here is that I wanted to jsonize results, but because of the way how result classes hierarchy is designed is does not work at all. See all the IllegalStateExceptions here: https://github.com/keenlabs/KeenClient-Java/blob/master/query/src/main/java/io/keen/client/java/result/QueryResult.java#L61

Personally I used to use lombok for generate boilerplate, but I don't want to add lombok dependency here. I'll consider using ReflectionToStringBuilder

Reply:
suggestion applied